content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -28,6 +26,15 @@ containers, the timezone set for the kube-controller-manager container determine
 that the cron job controller uses.
 {{< /caution >}}
 
+{{< caution >}}
+CronJob does not officially support setting timezone as explained above.
+
+Setting variables such as `CRON_TZ` is not officially supported.
```

Comment:
```suggestion
The [v1 CronJob API](/docs/reference/kubernetes-api/workload-resources/cron-job-v1/) lets
you set a timezone for a CronJob, by using the `CRON_TZ` variable in the time specification.

Setting variables in the time specification, including `CRON_TZ`, is not officially supported
by the Kubernetes project.
```

Reply:
That's not correct, the fact that this slipped through the cracks doesn't make this part of official API, it never was. But I did apply some of your suggestions, thx :)